Services.Test/InternalDevicePropertiesTest.cs

```diff
@@ -0,0 +1,222 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Models;
+using Services.Test.helpers;
+using Xunit;
+using Xunit.Abstractions;
+using static Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Models.DeviceModel;
+
+namespace Services.Test
+{
+    public class InternalDevicePropertiesTest
+    {
+        private InternalDeviceProperties target;
+
+        public InternalDevicePropertiesTest(ITestOutputHelper log)
```

Comment:
Not used for anything in this test in particular. Used throughout the tests in Simulation so the tests can log if needed.

Reply:
it's a convention to inject the test logger in all our tests, in case we need to log. Please leave it. It might be useful to add a comment explaining how to use it (across all our tests, in a separate PR)